core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -338,6 +338,7 @@ LOCAL_FUNC enable_mmu , : , .identity_map
 	ret
 END_FUNC enable_mmu
 
+	.balign	8
```

Comment:
Why is it important?

Reply:
If alignment is wrong we could get an alignment error. I'm I bit surprised that I didn't catch it as an error before, now I just took by inspection (I had an alignment issue on a 8 bytes variable in another PR).

Reply:
Ah, right, it is a data structure. For this commit:
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`